### DISCUSSION TITLE
Download deployment files

### DISCUSSION THREAD

**Discussion by FMJansen** [vercel/community#4223] [Help] [answered]
### Summary

Hey! I’ve got a deployed app. The API which I used to build the app two years ago doesn’t work anymore, so I can’t rebuild the app. Is there a way to **download the generated files (the output)** from Vercel? At https://vercel.com/{user}/{project}/{deployment}/source I can see source files, but not _output_ files. Also, there’ll be about 1600 individual pages to download, so I’m hoping there is an export function.

### Steps to Reproduce

Try to download the output files

**Comment by swarnava** [comment]
Hi there,

Due to security reason, source file cannot be downloaded and we don't store locally. We advise user to use own VCS instead.

**Reply by FMJansen** [reply]
Hey! Thanks for the reply, I feel it's not the answer to my question though. I'm not looking for the source, but the output files. 

The source files are still in my repository safe and sound; but building the output with them doesn't work anymore.

**Reply by FMJansen** [reply]
@swarnava Hey! Is there any way to re-open this question as it isn’t answered yet? Or should I create a new question?

**Reply by swarnava** [reply]
Hi, we don't allow export/download file from Vercel unfortunately.

**Reply by FMJansen** [reply]
Ah, that’s a pity. Thanks anyway! I’ll create a new answer to mark as the actual answer to prevent confusion.

**Comment by FMJansen** [comment, marked as answer]
As mentioned by @swarnava in the thread above; also the _output_ files can’t be downloaded or exported.
